roles/openshift_loadbalancer/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Set haproxy frontend port
+  openshift_facts:
+    role: loadbalancer
+    local_facts:
+      frontend_port: "{{ openshift_master_api_port | default(None) }}"
+
+- name: Set loadbalancer facts
+  openshift_facts:
+    role: loadbalancer
+    local_facts:
+      limit_nofile: "{{ openshift_loadbalancer_limit_nofile | default(None) }}"
+      default_maxconn: "{{ openshift_loadbalancer_default_maxconn | default(None) }}"
+      global_maxconn: "{{ openshift_loadbalancer_global_maxconn | default(None) }}"
+      frontends:
```

Comment:
Any objections to adding support for externally provided additional frontend and backends? I'm thinking something like openshift_loadbalancer_additional_{frontends,backends}, and then we could set frontends and backends to the union of the user provided values and the values we configure.